database/db.go

```diff
@@ -66,12 +68,17 @@ func (d dbLogger) AfterQuery(q *pg.QueryEvent) {
 
 // New returns a new DB object which wraps a connection to the database specified in config
 func New(config DBConfig) DB {
-	db := pg.Connect(&pg.Options{
+	options := &pg.Options{
 		Addr:     config.Address,
 		User:     config.User,
 		Database: config.Database,
 		Password: config.Password,
-	})
+	}
+	if config.EnableTLS {
+		*options.TLSConfig = tls.Config{}
```

Comment:
we don't need to provide anything to the tls config?

Reply:
Like I found in Redis... simply adding and empty config is likely enough to kick this into gear...

Reply:
yup ,basically tls.Config not being nil means that it should use tls.